src/app/app-routing.module.ts

```diff
@@ -1,8 +1,17 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
```

Comment:
You will also need to remove all references of these components form other parts of the app.